### PR TITLE
:sparkles: create page account fix #166

### LINKED ISSUE #166: Create page account

### PR DESCRIPTION
**Pull request by issa-diallo**

### CODE REVIEW

File: src/pages/AccountPage.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import { Container, Card, Badge } from 'react-bootstrap'
+import { useNavigate } from 'react-router-dom'
+import useTranslationHook from '../hook/TranslationHook'
+
+const AccountPage = (): JSX.Element => {
+  const navigate = useNavigate()
+  const { t } = useTranslationHook()
+
+  const goToProfile = (): void => {
+    navigate('account/profile')
```
Comment (AndreMiras) [inline src/pages/AccountPage.tsx:12]: Broken link, you're probably missing the leading slash

File: src/pages/AccountPage.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  return (
+    <>
+      <Container>
+        {/* ToDo add profil-header__avatar */}
+        <div className='profil-header__body'>
+          <h1 className='h1'>{t('title.myAccount')}</h1>
+          <p className='hstack'>
+            <Badge bg='secondary'>{t('text.standardAccount')}</Badge>
+          </p>
+        </div>
```
Comment (AndreMiras) [inline src/pages/AccountPage.tsx:23]: Do you really want an avatar, why would we want that?

File: src/pages/AccountPage.tsx [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  const { t } = useTranslationHook()
+
+  const goToProfile = (): void => {
+    navigate('account/profile')
+  }
+  return (
+    <>
+      <Container>
+        {/* ToDo add profil-header__avatar */}
+        <div className='profil-header__body'>
```
Comment (AndreMiras) [inline src/pages/AccountPage.tsx:18]: typo s/profil/profile/

Comment (issa-diallo) [reply src/pages/AccountPage.tsx:18]: deleted

File: src/pages/AccountPage.tsx (hunk 2 above)
Comment (issa-diallo) [reply src/pages/AccountPage.tsx:23]: Not necessarily

File: src/pages/AccountPage.tsx (hunk 1 above)
Comment (issa-diallo) [reply src/pages/AccountPage.tsx:12]: yes resolved

**Review by AndreMiras — APPROVED** [review]

**Pull request merged**